metadata.md

```diff
@@ -28,58 +38,87 @@ There may also be any number of delegated target roles.
 
 Signed by: Root role.
 
-Specifies the other top-level roles. When specifying these roles, the trusted keys for each role are listed along with the minimum number of those keys which are required to sign the role's metadata. We call this number the signature threshold.
+Specifies the other top-level roles. When specifying these roles, the trusted
+keys for each are listed, along with the minimum number of those keys required
+to sign the role's metadata. We call this number the signature threshold.
 
 See [example](https://raw.githubusercontent.com/theupdateframework/tuf/develop/tests/repository_data/repository/metadata/root.json) of Root metadata.
 
 ## Targets Metadata (targets.json)
 
 Signed by: Targets role.
 
-The targets.json metadata file lists hashes and sizes of target files. Target files are the actual files that clients are intending to download (for example, the software updates they are trying to obtain).
+Target files are the actual files that client wants to download, such as
```

Comment:
> Target files are the actual files **that client** wants to download, ...

Is this missing a word, or should it be "clients"?

Reply:
Fixed it. Should have been "actual files a client wants to download..."

Reply:
I went the plural to avoid a he/she thing. It's grammatically correct now.